packages/firebaseui-react/tests/unit/components/button.test.tsx

```diff
@@ -62,4 +62,28 @@ describe("Button Component", () => {
 
     expect(button).toBeDisabled();
   });
+
+  it("renders as a Slot component when asChild is true", () => {
```

Comment:
Do we need a test for when it is false?

"renders as a Slot component when asChild is false"